src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -94,23 +93,44 @@ class SoftmaxRegression
                     const double lambda = 0.0001,
                     const bool fitIntercept = false,
                     OptimizerType optimizer = OptimizerType());
-
+    /**
+   * Construct the SoftmaxRegression class with the provided data and labels.
+   * This will train the model. Optionally, the parameter 'lambda' can be
+   * passed, which controls the amount of L2-regularization in the objective
+   * function. By default, the model takes a small value.
+   *
+   * @tparam OptimizerType Desired optimizer type.
+   * @param data Input training features. Each column associate with one sample
+   * @param labels Labels associated with the feature data.
+   * @param inputSize Size of the input feature vector.
+   * @param numClasses Number of classes for classification.
+   * @param optimizer Desired optimizer.
+   * @param lambda L2-regularization constant.
+   * @param fitIntercept add intercept term or not.
```

Comment:
Missing the optimizer parameter.

Reply:
@zoq on line no. 107 ```   * @param optimizer Desired optimizer.``` is this?

Reply:
I see, I just looked at the last line, as the description order should match with the parameter order.